### PR TITLE
Fix repos count in profile

### PR DESCRIPTION
**Pull request by hosmelq**

### CODE REVIEW

File: src/components/user-profile.component.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const UserProfile = ({
           })}
       >
         <Text style={styles.unitNumber}>
-          {user.public_repos}
+          {user.public_repos + (user.total_private_repos || 0)}
```
Comment (housseindjirdeh) [inline src/components/user-profile.component.js]: Just want to ask - is `total_private_repos` undefined when a user has no private repos? Or does it default to 0?

Comment (hosmelq) [inline src/components/user-profile.component.js]: Yes, it's undefined

**Comment by hosmelq** [comment]
Done

**Comment by housseindjirdeh** [comment]
:tada: :tada: :tada:

**Pull request merged**